distest/interface.py

```diff
@@ -72,6 +72,9 @@ def __init__(
         """
         return await self.channel.send(content)
 
+    async def checkMessage(self, message):
```

Comment:
Name more specifically based on what the check is actually for (channel and author, not message content)